docs/_config.yml

```diff
@@ -6,3 +6,5 @@ url: https://UCL-ARC.github.io/python-tooling
 
 aux_links:
   Source repository: https://github.com/UCL-ARC/python-tooling
+
+logo: "https://raw.githubusercontent.com/UCL-ARC/python-tooling/main/images/logo.png"
```

Comment:
I couldn't get a relative path to work locally so used full URL - not sure if we could put `images` somewhere under `docs` directory to get this working and/or if there are relevant Jekyll / theme configuration options.

Reply:
Possibly the images are not included in the artifact upload step (therefore not included in the website deployment??).

I'd say this way is probably fine.

Reply:
I think you have to do this kind of thing for relative links https://github.com/paddyroddy/paddyroddy.github.io/blob/347e094cb0af0c609394b2beaa90770929feca1e/running.html#L14.

I think your approach is fine, though.

Reply:
Thanks useful to know for future reference!